packages/sw-lib/test/utils/dev-server.js

```diff
@@ -0,0 +1,8 @@
+const swTestingHelpers = require('sw-testing-helpers');
+const path = require('path');
+
+const testServer = new swTestingHelpers.TestServer();
+const serverPath = path.join(__dirname, '..', '..', '..', '..');
+testServer.startServer(serverPath, 8080).then((portNumber) => {
+ console.log(`http://localhost:${portNumber}/`);
```

Comment:
This is not used beyond manual testing.

Basically the test server used here adds the serviceworker allowed header that you need for local testing - one of the myriad of things I'd like to fix in my totes amaze testing tool ;)